packages/react/__tests__/element.js

```diff
@@ -0,0 +1,126 @@
+// @flow
```

Comment:
The added tests focus on the implementation details (`serializeStyles` calls etc). It would be IMHO much better to just test what you have reported and what you have reproed in [this codesandbox](https://codesandbox.io/s/emotion-issue-template-forked-ettgx?file=/src/index.js). The test would look like this:
1. render with `css={false}`
2. render with `css={() => {}}`
3. assert that it didn't crash

Reply:
kk thanks for the feedback I will address tonight.